php/plan.sh

```diff
@@ -18,19 +18,21 @@ pkg_deps=(
   core/libjpeg-turbo
   core/libpng
   core/openssl
+  core/readline
   core/zlib
 )
 pkg_build_deps=(
   core/bison2
   core/gcc
   core/make
   core/re2c
-  core/readline
 )
 pkg_bin_dirs=(bin sbin)
 pkg_lib_dirs=(lib)
 pkg_include_dirs=(include)
 pkg_interpreters=(bin/php)
+pkg_exports=([port]=port)
```

Comment:
Can we add the 'listen' parameter also here. If the user is binding to localhost, checking the listen parameter is the only way to know that.

Reply:
Easily added.